js/main.js

```diff
@@ -1,3 +1,51 @@
+const NAMES = [
+  'Морти',
+  'Шаман',
+  'Каспер',
+  'Миха',
+  'Вайс',
+  'Грей',
+];
+
+const MESSAGES = [
+  'Всё отлично!',
+  'В целом всё неплохо. Но не всё.',
+  'Когда вы делаете фотографию, хорошо бы убирать палец из кадра. В конце концов это просто непрофессионально.',
+  'Моя бабушка случайно чихнула с фотоаппаратом в руках и у неё получилась фотография лучше.',
+  'Я поскользнулся на банановой кожуре и уронил фотоаппарат на кота и у меня получилась фотография лучше.',
+  'Лица у людей на фотке перекошены, как будто их избивают. Как можно было поймать такой неудачный момент?!',
+];
+
+const DESCRIPTIONS = [
+  'Отдыхаю с котиком.',
+  'Мой песель самый лучший',
+  'Морти - собака сутулая',
+  'Не нравится - не смотри!',
+  'Жизнь - штука тяжелая!',
+  'Покойтесь с миром обои...',
+];
+
+const getRandomArrayElement = (elements) => elements[randomNumber(0, elements.length - 1)];
+
+function generatePhoto() {
+  return {
+    id: randomNumber(1,25),
+    url: `photos/${randomNumber(1,25)}.jpg`,
+    description: getRandomArrayElement(DESCRIPTIONS),
+    likes: randomNumber(15,200),
+    comment: {
+      id: randomNumber(1,200),
+      avatar: `img/avatar-${randomNumber(1,6)}.svg`,
+      message: getRandomArrayElement(MESSAGES),
+      name: getRandomArrayElement(NAMES)
+    }
+  };
+}
+
+const photoArray = Array.from({length: 25}, generatePhoto);
+
+photoArray();//чтобы не ругался eslint
```

Comment:
Такая комбинация вызовет ошибку `TypeError: photoArray is not a function.` Так как ты пытаешься вызвать функцию из константы. 
На счет уникальности могу предложит вот такой вариант. Через генерацию сиквенса (уникального значения). В качестве параметра в from мы передаем mapFn, из которой получаем номер итерации
```suggestion
const photoArray =  () => Array.from({length: 25}, (_,i) => generatePhoto(i));

photoArray();//чтобы не ругался eslint
```